src/config/php.md

```diff
@@ -0,0 +1,35 @@
+# PHP
+
+There are two ways to install PHP packages with XBPS:
+
+1. Using the versioned (otherwise known as pinned) packages (reccomended).
+2. Using the metapackages.
+
+## Pinned PHP Packages
```

Comment:
```suggestion
## Versioned/Pinned PHP Packages
```

Reply:
I like Versioned PHP packages more, pinned implies something like holding or repolocking it in the package manager to me.

Reply:
To be honest, that's also what I thought when I saw "pinned" being used here. It just seems out of place.